web/modules/custom/hoeringsportal_citizen_proposal/hoeringsportal_citizen_proposal.install

```diff
@@ -146,3 +146,31 @@ function hoeringsportal_citizen_proposal_update_9001(): void {
 
   $schema->createTable($table_name, $table_schema);
 }
+
+/**
+ * Changes citizen identifier storage.
+ */
+function hoeringsportal_citizen_proposal_update_9002(): void {
```

Comment:
I don't think this will run when the module is enabled, so i think we need to modify the hook_schema as well.